challenge.rb

```diff
@@ -1,26 +1,26 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map{|i| i.capitalize}
```

Comment:
Try out this syntax: `input.map(&:capitalize)`

Reply:
![mindblown](https://media.giphy.com/media/75ZaxapnyMp2w/giphy.gif)